eng/native/configurecompiler.cmake

```diff
@@ -551,8 +551,8 @@ if (MSVC)
   # /W3 is added by default by CMake, so remove it
   string(REPLACE "/W3" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
   string(REPLACE "/W3" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
-  # set default warning level to 3 but allow targets to override it.
-  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/W$<GENEX_EVAL:$<IF:$<BOOL:$<TARGET_PROPERTY:MSVC_WARNING_LEVEL>>,$<TARGET_PROPERTY:MSVC_WARNING_LEVEL>,3>>>)
+  # set default warning level to 4 but allow targets to override it.
+  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/W$<GENEX_EVAL:$<IF:$<BOOL:$<TARGET_PROPERTY:MSVC_WARNING_LEVEL>>,$<TARGET_PROPERTY:MSVC_WARNING_LEVEL>,4>>>)
```

Comment:
Do you want to add that `# [[! Microsoft.Security.SystemsADM.10086 !]]` marker while you are in here?

Reply:
What is that?

Reply:
Nevermind, I found it.

Reply:
https://github.com/dotnet/runtime/issues/66154 says to include `Microsoft.Security.SystemsADM.10086` in the file configuring the compilation to help with auditing. That specific string is from the example for adding it to a cmake file.